connectors/src/connectors/salesforce/index.ts

```diff
@@ -86,87 +150,175 @@ export class SalesforceConnectorManager extends BaseConnectorManager<null> {
     throw new Error("Not implemented");
   }
 
-  async sync(): Promise<Result<string, Error>> {
-    // TODO(salesforce): implement this
-
-    throw new Error("Not implemented");
+  async sync({
+    fromTs,
+  }: {
+    fromTs: number | null;
+  }): Promise<Result<string, Error>> {
+    logger.info({ fromTs }, "To be implemented");
```

Comment:
should we keep the TODO ?

Reply:
Indeed, I'll remove it on the second PR you reviewed, to avoid the rebase!

Reply:
Wait, actually it will be probably implemented for the free text rag part, so keeping it!